lib/deployment_runner.py

```diff
@@ -0,0 +1,157 @@
+import yaml
```

Comment:
I would put a header comment here that describes the purpose of this library, and how it basically works.